src/components/NewUser.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="container">
+    <b-form @submit="updateUser">
+      <div class="text-left">
+        <b-form-group
+          label-cols-sm="3"
+          label="Handle:"
+          label-align-sm="right"
+          label-for="userName"
+        >
+          <b-input id="userName" v-model="currentUser.name"></b-input>
+        </b-form-group>
+        <b-form-group
+          label-cols-sm="3"
+          label="Email Address:"
+          label-align-sm="right"
+          label-for="userEmail"
+        >
+          <b-input id="userEmail" v-model="currentUser.email"></b-input>
+        </b-form-group>
+        <b-form-group
+          label-cols-sm="3"
+          label="Receive Emails:"
+          label-align-sm="right"
+          class="mb-0"
+          label-for="userReceiveEmails"
+        >
+          <b-form-radio-group
+            class="pt-2"
+            id="userReceiveEmails"
+            name="receiveEmails"
+            v-model="currentUser.receiveEmails"
+          >
+            <b-form-radio value="true">Yes</b-form-radio>
+            <b-form-radio value="false">No</b-form-radio>
+          </b-form-radio-group>
+        </b-form-group>
+        <div class="text-center">
+          <b-button type="submit">Save</b-button>
+        </div>
+      </div>
+    </b-form>
+    <ServerInputErrors
+      :graphError="graphError"
```

Comment:
Passing the graphError data variable to the ServerInputErrors component using the graphError prop